lib/results.js

```diff
@@ -157,7 +157,7 @@ function encodeResult (res, count) {
         output += inner + 'at: ' + res.at + '\n';
     }
 
-    var actualStack = res.actual && res.actual.stack;
+    var actualStack = res.actual !== undefined && res.actual.stack;
```

Comment:
this will cause it to throw if `res.actual` is `null`.

If `res.actual` is a primitive (even a truthy primitive), `.stack` will never have a meaningful value.

This check should probably really be `res.actual && (typeof res.actual === 'object' || typeof res.actual === 'function') ? res.actual.stack : null`

Reply:
I'm working slightly retarded here, on windows a lot of the unit tests fail. Will update the pr.

Reply:
Oh, I see. It was working on my machine for the wrong reason. The actual bug is:

when res.actual === false,
res.actual && res.actual.stack === false, not undefined

Reply:
@nhamer fyi, i edited your comment to remove the inappropriate word.